File: java/src/main/java/simple/osmdroid/egorc/com/java/MainActivity.java
package simple.osmdroid.egorc.com.java;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import org.osmdroid.tileprovider.tilesource.XYTileSource;
import org.osmdroid.util.GeoPoint;
import org.osmdroid.views.MapView;

public class MainActivity extends AppCompatActivity {

    private GeoPoint CHELAYBINSK = new GeoPoint(55.160928, 61.414103);

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        MapView mapView = findViewById(R.id.map_view);


        mapView.setBuiltInZoomControls(true);


        mapView.setMultiTouchControls(true);
        mapView.setTileSource(new XYTileSource(
                "Hikebikemap",
                1,
                15,
                256,
                ".png",
                new String[]{}));
        mapView.setUseDataConnection(false);
        mapView.getController().setCenter(CHELAYBINSK);
        mapView.getController().setZoom(15);
    }
}

File: kotlin/src/main/java/com/egorc/osmdroid/simple/MainActivity.kt
package com.egorc.osmdroid.simple

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import org.osmdroid.tileprovider.tilesource.XYTileSource
import org.osmdroid.util.GeoPoint

class MainActivity : AppCompatActivity() {

    private val CHELAYBINSK = GeoPoint(55.160928, 61.414103)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        map_view.setBuiltInZoomControls(true)
        map_view.setMultiTouchControls(true)
        map_view.setTileSource(XYTileSource(
                "Hikebikemap",
                7,
                17,
                256,
                ".png",
                arrayOf()
        ))
        map_view.setUseDataConnection(false)
        map_view.controller.setCenter(CHELAYBINSK)
        map_view.controller.setZoom(15)
    }
}
